third_party/llvm-openmp/BUILD

```
@@ -14,17 +13,17 @@ load(
 )
 
 genrule(
-    name = "il8n_id",
+    name = "kmp_il8n_id",
```

I meant to say the correct version should be `18` (one eight) and not `l8` (L eight). Sorry that you have to fix this again!
```suggestion
    name = "kmp_i18n_id",
```